app/app.go

```diff
@@ -6,12 +6,14 @@ import (
 	"path/filepath"
 
 	authsims "github.com/cosmos/cosmos-sdk/x/auth/simulation"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
```

Comment:
@shellvish if you have time, might make sense for you to review this file as well!